README.md

```diff
@@ -14,8 +14,10 @@ The following commands are provided by the plugin:
 * `odk:subset`: to create ontology subsets;
 * `odk:check-align`: to check the alignment of an ontology against an
   upper-level ontology and/or arbitrary root terms;
-* `odk:normalize`: to “normalise” an ontology,
-* `odk:import`: to add/remove import declarations.
+* `odk:normalize`: to "normalise" an ontology;
+* `odk:import`: to add/remove import declarations;
+* `odk:obsolete-replace`: to obsolete an entity and replace all its usages
```

Comment:
It’d be nice (though I will not _demand_ that before I accept the PR) to have even a basic documentation page for this command, like we have for the commands (e.g. [odk:check-align](https://incatools.github.io/odk-robot-plugin/alignment.html)). The sources for the documentation are there: https://github.com/INCATools/odk-robot-plugin/tree/main/src/site/markdown.